wiki/People/Beatmap_Nomination_Group/en.md

```diff
@@ -9,33 +9,33 @@
 [Ym]: /wiki/shared/Insane-m.png "Yes"
 
 Beatmap Nomination Group
-=========================
+========================
 
-The **Beatmap Nomination Group**, more commonly referred to as BN, is a group in charge of [beatmap nomination](/wiki/Beatmaps).
+The **Beatmap Nomination Group**, more commonly referred to as the BNG, is a group in charge of [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure). Individual members are commonly referred to as BNs, or Beatmap Nominators.
 This group has undergone many changes recently.
-As of August 22, 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums.
-The most recent change was on February 1, 2015 when the team was changed from Beatmap Appreciation Team to Beatmap Nominator Group.
+As of 22. August 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums.
+The most recent change was on 01. February 2015 when the team was changed from Beatmap Appreciation Team to Beatmap Nominator Group.
 
 Roles and Responsibilities
 --------------------------
 
 ![The Nominator Badge](Nominator.png "The Nominator Badge")
 
 Members of the **Beatmap Nomination Group** are responsible primarily for [modding](/wiki/Modding) the [pending beatmaps](/wiki/Beatmaps) into the ranking cycle.
-BN are the people to go to for any modding or advice requests about anything mapping or modding related.
+BNs are the people to go to for any modding or advice requests about anything mapping or modding related.
 
 Becoming a BN
 -------------
 
-With the change to the team on August 22, 2014, joining the BN is now a lot easier than it used to be.
-You can read [this thread](https://osu.ppy.sh/forum/t/471344) on how to join the Beatmap Nomination Group.
+With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be.
+You can read [How to join the BNG](https://osu.ppy.sh/forum/t/471344) on how to join the Beatmap Nomination Group.
```

Comment:
Rephrasing suggestion:
`For more details, see the forum post [How to join the BNG](https://osu.ppy.sh/forum/t/471344) on how to join the Beatmap Nomination Group.`